elasticdl/python/pserver/embedding_table.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import tensorflow as tf
+
+
+class EmbeddingTable(object):
```

Comment:
May add some comments to explain the class.

Reply:
Done

Reply:
We may add `Parameters` class in the same file. If so, we should rename this file, such as `datatypes.py` or `ps_datatype.py.

Also, I used the directory name `ps` in another PR. We can use `pserver` in the proto service definition, and use a same directory.

Reply:
It's better to keep each module short and clear. The `Parameters` class will be defined in another file `parameters.py`

And maybe `ps` directory is too short? `pserver` is a little more clear, I could move the main.py to pserver directory, to make all related file in the same directory.

Reply:
Using one file or two files for datatypes, both are fine in this case. 

For the directory name, as long as the meaning is clear, short name is fine.  Also consistent with design doc.

Reply:
Done. make it consistent with the design doc